content/grants/apprenticeships/ua-heads-up-poker.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "Apprenticeship: Heads-up poker"
+title = "[Closed] Userspace Apprenticeship: Heads-Up Poker"
```

Comment:
No need to write in `[Closed]` to the title—just set the `assignee` property to the `@p` of the person(s) that are working on it and it'll no longer be claimable.